app/actions/internetConnectivityPromptActions.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import { createActions } from 'spunky'
+
+export const ID = 'internet-connectivity-prompt'
+
+export const internetConnectionPromptPresented = createActions(
+  ID,
+  (bool = false) => (): {
+    internetConnectionPromptPresented: boolean,
+  } => ({
+    internetConnectionPromptPresented: bool,
+  }),
+)
```

Comment:
A simple boolean being added to state to determine whether or not the Dashboard should redirect to the offline prompt